rust-toolchain.toml

```diff
@@ -5,4 +5,4 @@
 # This is the version that will be used in CI when building
 # shadow, running lints, etc.
 [toolchain]
-channel = "1.66.0"
+channel = "1.66.1"
```

Comment:
While we're editing this file, maybe remove the above comment since this version is used for shadow in all contexts, not just the CI?

Reply:
Done